src/test/java/features/RunCucumberTest.java

```diff
@@ -0,0 +1,6 @@
+package features;
+
+import io.cucumber.junit.platform.engine.Cucumber;
+
+@Cucumber
+public class RunCucumberTest {}
```

Comment:
We should rename this class to `CucumberTest` so we avoid having class names that are actions.

Reply:
No the database is not prepopulated with entries, Its made such  that it inserts in the database before it gets, etc....